linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/smartcard_pam_enabled/ansible/shared.yml

```diff
@@ -4,29 +4,40 @@
 # complexity = low
 # disruption = low
 
+{{%- if 'sle' in product %}}
```

Comment:
I know the Playbook was already there and not aligned to the Project Style Guide, but it would be great if you already update it since you are working on it.

[Projetc Style Guide](https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#ansible) for Ansible Playbooks

Reply:
Thanks